src/lib/api/container/client/containerApiClient.client.test.ts

```diff
@@ -120,4 +120,41 @@ describe('containerApiClient', () => {
       expect(result).toEqual(Err(mockError.message));
     });
   });
+
+  //test for creating new container function
+  describe('postContainer', () => {
+    /* Arrange common mock data */
+    const mockRequest = { groupId: 'groupId', name: 'newContainer' };
```

Comment:
It's very very minor thing tho, how about renaming variable `mockRequest` to  `mockRequestBody`?
Because the variable in here is not the request itself, but the request body. It'll make the code a bit easier to understand !

Reply:
@kanta1207 I see your point. For readality sake, right!? I will update it this one also!